### DISCUSSION TITLE
求教如何在自定义运行脚本中透传参数

### DISCUSSION THREAD

**Discussion by MasterBalll** [xmake-io/xmake#3382] [Q&A] [answered]
比如在运行 `xmake r <target> -i 123 -o 456` 时，想从 `on_run` 里原样获得 `-i 123 -o 456` ，如何做到？

**Comment by star-hengxing** [comment, marked as answer]
https://github.com/xmake-io/xmake/blob/master/xmake/actions/run/main.lua#L57

**Reply by MasterBalll** [reply]
噢噢 感谢感谢
